docs/meson.build

```diff
@@ -43,6 +43,18 @@ if gidocgen_dep.found() and docs_python_deps.allowed() and gidocgen_app.found()
     install_dir: join_paths(datadir, 'doc'),
   )
 
+  hsi_md = custom_target('generate-hsi-spec',
+    input : 'hsi.md',
```

Comment:
I know it's not required with meson, but I think it makes sense to rename this `hsi.md.in` so that this doesn't render in github U/I anymore.  Since it's missing all the chunks of data it's not very useful in this form.

Another idea is to have a comment that is here and removed by the python script:

"Some information in this specification is dynamically built.  Visit https://fwupd.github.io/libfwupdplugin/hsi.html for the complete specification."

Reply:
I think `hsi.md.in` makes sense. Fixed.